Sources/Commands/SwiftTestCommand.swift

```diff
@@ -130,6 +130,11 @@ struct TestEventStreamOptions: ParsableArguments {
     @Option(name: .customLong("attachments-path"),
             help: "Path where attachments should be written (Swift Testing only). This path must be an existing directory the current user can write to. If not specified, any attachments created during testing are discarded.")
     var attachmentsPath: AbsolutePath?
+
+    /// If experimental features should be enabled..
+    @Option(name: .customLong("enable-experimental-features"),
+            help: .private)
+    var enableExperimentalFeatures: Bool
```

Comment:
Offline, I made a suggestion that we scope this flag more narrowly, to only apply to one feature (singular instead of plural) and accept the name of that feature as an argument. The problem with having a single, catch-all “enable all the experimental features” flag is the _next_ time someone adds some new experimental thing, if someone was already passing the flag then they will break.